pkg/scraper/url.go

```diff
@@ -157,6 +159,11 @@ func urlFromCDP(ctx context.Context, urlCDP string, driverOptions scraperDriverO
 				chromedp.UserDataDir(dir),
 				chromedp.ExecPath(cdpPath),
 			)
+			if globalConfig.GetProxy() != "" {
+				url, _, _ := splitProxyAuth(globalConfig.GetProxy())
+				opts = append(opts, chromedp.ProxyServer(url))
+			}
+
```

Comment:
Doing this here means that the proxy is set only for local CDP instances (using paths) and not remote.
From a quick look it doesnt seem that there is a way to do it for remote allocators via cdp (which  seems fair considering it needs to be done when starting the chromium instance and remote instances are already running)
Imho this is good enough as this is a limitation of the API, we should probably add some more documentation mentioning the use cases / limitations though?

For example i run the headless-shell container like this
```bash
docker run -d -p 9222:9222 --rm --name headless-shell chromedp/headless-shellchromedp/headless-shell
```
and use a local proxy eg `http://192.168.1.2:1080`
With latest PR i can access CDP but as mentioned nothing goes through the proxy when using it
I had to force the use of a proxy when running the container
```bash
docker run -d -p 9222:9222 --rm --name headless-shell -e "HTTP_PROXY=http://192.168.1.2:1080" -e "HTTPS_PROXY=http://192.168.1.2:1080" chromedp/headless-shell
```
to make the traffic go through the proxy (this has the limitation of authenticated proxies not being supported, for those cases one would need to run a local proxy that handles auth internally)

Reply:
That is correct, CDP itself needs to be configured to use a Proxy independently. Just like your own browser.
In this context we can only configure the Proxy for stash-backend.
I guess this should probably be clarified in the CDP documentation ? Not sure where this lives.

Reply:
I was thinking in the in-app documentation <https://github.com/stashapp/stash/blob/develop/ui/v2.5/src/docs/en/Manual/>
Probably a few stuff (for plain proxy and CDP) here <https://github.com/stashapp/stash/blob/develop/ui/v2.5/src/docs/en/Manual/Configuration.md#scraping>  and then we can update the docs site with more details/examples?

Reply:
I added a bit of documentation in-app, it's lightweight, to fit the amount of documentation for other features.
Considering CDP fits in 2-3 lines, I haven't added anything to it, would add unnecessary complexity.

I'm not sure how documentation is going to work in Stash in the future, what goes in-app and what goes to Stash-Docs. For now I think we can stop here, and discuss more on discord #documentation for the future